### ISSUE TITLE
接続の"である"

### ISSUE THREAD

**Issue by azu** [azu/textlint-rule-no-mix-dearu-desumasu#10]
"ではなく"とかはさすがにデフォルトでは厳しい。false positiveな感じがする

> 動的にメソッドを追加するだけ**で**はなく、既存の実装を上書きするmonkey patchのようなものもプラグインとして追加することができます。

```
    {
        "word_id": 3168910,
        "word_type": "KNOWN",
        "word_position": 11,
        "surface_form": "する",
        "pos": "動詞",
        "pos_detail_1": "自立",
        "pos_detail_2": "*",
        "pos_detail_3": "*",
        "conjugated_type": "サ変・スル",
        "conjugated_form": "基本形",
        "basic_form": "する",
        "reading": "スル",
        "pronunciation": "スル"
    },
    {
        "word_id": 2594860,
        "word_type": "KNOWN",
        "word_position": 13,
        "surface_form": "だけ",
        "pos": "助詞",
        "pos_detail_1": "副助詞",
        "pos_detail_2": "*",
        "pos_detail_3": "*",
        "conjugated_type": "*",
        "conjugated_form": "*",
        "basic_form": "だけ",
        "reading": "ダケ",
        "pronunciation": "ダケ"
    },
    {
        "word_id": 305030,
        "word_type": "KNOWN",
        "word_position": 15,
        "surface_form": "で",
        "pos": "助動詞",
        "pos_detail_1": "*",
        "pos_detail_2": "*",
        "pos_detail_3": "*",
        "conjugated_type": "特殊・ダ",
        "conjugated_form": "連用形",
        "basic_form": "だ",
        "reading": "デ",
        "pronunciation": "デ"
    },
```

**Comment by azu** [comment]
> このパターンだけでは難しい部分が出てくると思います

```js
    {
        "word_id": 305030,
        "word_type": "KNOWN",
        "word_position": 9,
        "surface_form": "で",
        "pos": "助動詞",
        "pos_detail_1": "*",
        "pos_detail_2": "*",
        "pos_detail_3": "*",
        "conjugated_type": "特殊・ダ",
        "conjugated_form": "連用形",
        "basic_form": "だ",
        "reading": "デ",
        "pronunciation": "デ"
    },
```

**Comment by azu** [comment]
[3.0.4](https://github.com/azu/analyze-desumasu-dearu/releases/tag/3.0.4 "3.0.4") で厳密な方向へ変更
